gc/base/EnvironmentBase.hpp

```diff
@@ -150,6 +150,8 @@ class MM_EnvironmentBase : public MM_BaseVirtual
 
 	uintptr_t approxScanCacheCount; /**< Local copy of approximate entries in global Cache Scan List. Updated upon allocation of new cache. */
 
+	uintptr_t depthCount; /**< Used for dynamic breadth first scan ordering. Counter for the current copying depth based on the initial object copied. */
```

Comment:
give it a bit more specific name: hotField(Copy?)DepthCount. Since we hope to be used in Balanced copy-forward too, it should not be just Scavenger specific name.